tools/tune/eq/example_fir_eq.m

```diff
@@ -21,7 +21,7 @@ function example_fir_eq()
 %% -------------------
 %% Example 1: Loudness
 %% -------------------
-fn.bin = 'eq_fir_loudness.bin';
+fn.bin = 'eq_fir_loudness.blob';
```

Comment:
Thanks, I was a bit confused by `.bin`. It's much clearer now with `.blob`!

:-)))